Common/Enums/FilteringFailReason.cs

```diff
@@ -1,43 +1,51 @@
 namespace Common.Enums;
 
 /// <summary>
-/// Explains why the player failed filtering
+/// Indicates why the player failed filtering
 /// </summary>
 [Flags]
 public enum FilteringFailReason
 {
     /// <summary>
-    /// The player passed filtering, thus there is no failure reason
+    /// The player does not exist in the system at all
```

Comment:
```suggestion
    /// The player is not indexed by o!TR
```
reads a bit nicer maybe?